src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -90,13 +93,42 @@ public void focus() {
     }
 
     /**
-     * Copies the URL to the user guide to the clipboard.
+     * Opens the User Guide in the Browser.
+     * Solution referenced from https://stackoverflow.com/a/54869038/16777554
      */
     @FXML
-    private void copyUrl() {
-        final Clipboard clipboard = Clipboard.getSystemClipboard();
-        final ClipboardContent url = new ClipboardContent();
-        url.putString(USERGUIDE_URL);
-        clipboard.setContent(url);
+    private void openInBrowser() {
+        logger.fine("URL to open: " + USERGUIDE_URL); // change to fine logging
+        String oS = System.getProperty("os.name").toLowerCase();
+        logger.fine("Operating system detected: " + oS);
+
+        try {
+            if (Desktop.isDesktopSupported()) { // Windows
+                logger.fine("Using Desktop.browse on Windows to open webpage.");
+                Desktop desktop = Desktop.getDesktop();
+                desktop.browse(new URI(USERGUIDE_URL));
+
+            } else {
+                Runtime runtime = Runtime.getRuntime();
+
+                if (oS.contains("mac")) {
+                    logger.fine("Using 'open' on Mac to open webpage.");
+                    runtime.exec("open " + USERGUIDE_URL);
+
+                } else if (oS.contains("nix") || oS.contains("nux")) {
+                    logger.fine("Using 'xdg-open' on Linux to open webpage.");
+                    runtime.exec("xdg-open " + USERGUIDE_URL);
```

Comment:
not sure, but it might be a good idea to check the `exitValue()` when using `runtime.exec()` too

Reply:
Will log it in the fine level then. Thanks for the suggestion!